### PR TITLE
Ruby 2.7 Upgrade

### PR DESCRIPTION
**Pull request by mattcen**
Closes: #77

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -402,4 +402,4 @@ DEPENDENCIES
   webpacker
 
 BUNDLED WITH
-   1.17.3
+   2.1.4
```
Comment (cassar) [inline Gemfile.lock:405]: The changes to this file will (hopefully) be generated when you update the `.ruby-version` file and run `bundle install`,

Comment (mattcen) [reply Gemfile.lock:405]: Ah right, that makes sense. I'm learning how Ruby works as I go, so don't have the full picture of how everything fits together yet.

File: .env.example [hunk 2]
```diff
@@ -2,3 +2,14 @@ POSTGRES_PASSWORD=SuperSecretPass
...
+# To set to a particular a point in the competition run db:setup with the below
+# STAGE variables set.
+
+# PRE_CONNECTION - For before the connection events.
+# MID_COMPETITION - For mid way through the competition.
+# POST_COMPETITION - For After the competition.
+# MID_JUDGING - For during competition judging.
+# POST_JUDGING - For after competition judging.
+# POST_AWARDS - For after awards events.
+#STAGE=PRE_CONNECTION
```
Comment (cassar) [inline .env.example:15]: Nitpick: is this spacing correct?

**Review by cassar — APPROVED** [review]
Gave this a rebase and ran locally, suggest we don't merge until RVM is update on production and the new version of Ruby is installed.

File: .env.example (hunk 2 above)
Comment (mattcen) [reply .env.example:15]: Yes. Line 15 serves both as an example *and* is designed to be uncommented and actually used (possibly with a different value as per the above lines), lines 6-14 are designed to remain commented as documentation.

Comment (cassar) [reply .env.example:15]: Ahh nice, no worries. 👍

**Pull request merged**